README.md

```diff
@@ -402,7 +396,7 @@ to `./stage2` (if building a minimal system).
 
 ```bash
 # Example for building a lite system
-echo "IMG_NAME='Raspbian'" > config
+echo "IMG_NAME='Raspberry Pi OS'" > config
```

Comment:
debootstrap falls over when it sees spaces and IMG_NAME forms a part of the initial work path.